package.json

```diff
@@ -105,7 +104,8 @@
     "vite-plugin-svgr": "^3.2.0",
     "vite-tsconfig-paths": "~4.2.0",
     "vitest": "~0.32.0",
-    "ws": "8.9.0"
+    "ws": "8.9.0",
+    "@nx/eslint": "17.0.3"
```

Comment:
Might be worth sorting this because the next run of yarn will reorder this

Reply:
I'll merge and push a distinct commit for this.